frontend/src/components/resultPage/resultsReferredOut/ReferredOutTests.js

```diff
@@ -0,0 +1,564 @@
+
+  useEffect(() => {
+    if (!innitialized) {
+      // let updatedDate = encodeDate(configurationProperties.currentDateAsText);
+      let updatedDate = configurationProperties.currentDateAsText;
+      setReferredOutTestsFormValues({
+        ...referredOutTestsFormValues,
+        dateOfBirth: updatedDate,
+        startDate: updatedDate,
+        endDate: updatedDate,
+      });
+    }
+    if (referredOutTestsFormValues.dateOfBirth != "") {
+      setInnitialized(true);
+    }
+  }, [
+    referredOutTestsFormValues,
+    innitialized,
+    configurationProperties.currentDateAsText,
+  ]);
+
+  return (
+    <>
+      <PageBreadCrumb breadcrumbs={breadcrumbs} />
+      <Grid fullWidth={true}>
+        <Column lg={16}>
+          <Section>
+            <Section>
+              <Heading>
+                <FormattedMessage id="referral.out.head" />
+              </Heading>
+            </Section>
+          </Section>
+        </Column>
+      </Grid>
+
+      {notificationVisible === true ? <AlertDialog /> : ""}
+      {loading && <Loading />}
+      <div className="orderLegendBody">
+        <Grid fullWidth={true}>
+          <Column lg={16} md={8} sm={4}>
+            <Section>
+              <div className="formInlineDiv">
+                <h5>
+                  <FormattedMessage id="referral.main.button" />
+                </h5>
+              </div>
+              <br />
+              <SearchPatientForm
+                getSelectedPatient={getSelectedPatient}
+              ></SearchPatientForm>
+              <div className="formInlineDiv">
+                <div className="searchActionButtons">
+                  <Button type="button" onClick={handleReferredOutPatient}>
+                    <FormattedMessage
+                      id="referral.main.button"
+                      defaultMessage="Search Referrals By Patient"
+                    />
+                  </Button>
+                </div>
+              </div>
+            </Section>
+          </Column>
+        </Grid>
+        <hr />
+        <Formik
+          initialValues={referredOutTestsFormValues}
+          enableReinitialize={true}
+          // validationSchema={}
+          onSubmit={handleSubmit}
+          onChange
+        >
+          {({
+            values,
+            //errors,
+            //touched,
+            setFieldValue,
+            handleChange,
+            handleBlur,
+            handleSubmit,
+          }) => (
+            <Form
+              onSubmit={handleSubmit}
+              onChange={handleChange}
+              onBlur={handleBlur}
+            >
+              <Field name="guid">
+                {({ field }) => (
+                  <input type="hidden" name={field.name} id={field.name} />
+                )}
+              </Field>
+              <Grid fullWidth={true}>
+                <Column lg={16} md={8} sm={4}>
+                  <Section>
+                    <div className="inlineDiv">
```

Comment:
Can you just use the carbon Grid and Column components for the Form Layout instead of thecustom css classes. its easier to responsivenes see issue https://github.com/I-TECH-UW/OpenELIS-Global-2/issues/978